test/helpers/cmd.go

```diff
@@ -140,7 +141,7 @@ func (res *CmdRes) CombineOutput() *bytes.Buffer {
 
 // IntOutput returns the stdout of res as an integer
 func (res *CmdRes) IntOutput() (int, error) {
-	return strconv.Atoi(strings.Trim(res.stdout.String(), "\n"))
+	return strconv.Atoi(strings.Trim(strings.Trim(res.stdout.String(), "\n"), "\r"))
```

Comment:
AFAIK `strings.Trim` trims all characters from the string provided in the second argument, so I think that one call should be enough:

```
return trconv.Atoi(strings.Trim(res.stdout.String(), "\n\r"))
```